pyqir-generator/tests/test_conversions.py

```diff
@@ -13,22 +13,22 @@ def get_module() -> SimpleModule:
 
 def test_ir_round_trip_is_identical() -> None:
     expected_ir = get_module().ir()
-    bitcode = ir_to_bitcode(expected_ir, "test")
-    converted_ir = bitcode_to_ir(bitcode, "test")
+    bitcode = ir_to_bitcode(expected_ir, "test", None)
+    converted_ir = bitcode_to_ir(bitcode, "test", None)
     assert expected_ir == converted_ir
 
 
 def test_ir_round_trip_is_not_identical_when_module_name_isnot_supplied() -> None:
     expected_ir = get_module().ir()
-    bitcode = ir_to_bitcode(expected_ir)
-    converted_ir = bitcode_to_ir(bitcode)
+    bitcode = ir_to_bitcode(expected_ir, None, None)
+    converted_ir = bitcode_to_ir(bitcode, None, None)
```

Comment:
For this change, is the `None` required because of how type-checked this file is or will other python consumers now have to explicitly include `None` arguments? If it's the latter, then this may end up being a breaking change for things like qiskit to QIR test infra.

Reply:
This file wasn't type checked in CI before. I didn't change the type of `ir_to_bitcode` and `bitcode_to_ir` - they were already type hinted as having three required parameters (although apparently the implementation allows default arguments).

Reply:
I can change the type instead to allow default arguments, that probably looks nicer for callers.